dev/service-registry/service-locator/src/main/scala/com/lightbend/lagom/registry/impl/ServiceRegistryActor.scala

```diff
@@ -183,7 +183,7 @@ class ServiceRegistryActor @Inject() (unmanagedServices: UnmanagedServices) exte
     case Register(name, service) =>
       sender() ! registry.register(name, service)
       router.rebuild(registry)
-    case Lookup(serviceName, portName) => sender() ! registry.lookup(serviceName, portName)
+    case l @ Lookup(serviceName, portName) => sender() ! registry.lookup(serviceName, portName)
```

Comment:
We are not using `l`. Maybe a left-over of some other refactoring?

Reply:
Oops, yes. I had started to work on using akka discovery directly, but decided it would be better to break into a separate commit.